README.md

```diff
@@ -0,0 +1,61 @@
+# Playwright/Pytest & Python Test Automation Suite
+
+### About
+
+This project demonstrates skills in building a test automation suite from scratch, using Playwright/Pytest and Python. It is focused on [The Internet](https://the-internet.herokuapp.com/), a demo site made for automation practice by [Elemental Selenium](https://elementalselenium.com/). The suite covers a wide selection of the site's pages and functionalities.
+
+### Getting Started
+
+1. **Clone the repository:**
+
+   ```bash
+   git clone https://github.com/ctmcnutt/playwright-playground
+   ```
+
+2. **(Optional) Set up virtual environment**
+
+  ```bash
```

Comment:
Spacing on this block leaves it out of line with the rest of the bash example code around it. Should be indented to match

Reply:
Done